app/root.jsx

```diff
@@ -7,18 +7,306 @@ import {
   ScrollRestoration,
 } from "remix";
 import AlbyHeadIcon from "../public/images/alby_icon_head_icon.png";
+import CatamaranThin from "../public/fonts/Catamaran-Thin.woff2";
```

Comment:
I doubt that we require and use half of these font weights and subsets but cannot check now.